lighthouse-core/scripts/legacy-javascript/summary-signals.json

```diff
@@ -589,45 +357,33 @@
       "name": "core-js-3-only-polyfill/es6-array-from",
       "signals": "Array.from"
     },
-    {
-      "name": "core-js-3-only-polyfill/es6-array-index-of",
-      "signals": ""
-    },
     {
       "name": "core-js-3-only-polyfill/es6-array-is-array",
       "signals": "Array.isArray"
     },
-    {
-      "name": "core-js-3-only-polyfill/es6-array-iterator",
-      "signals": ""
-    },
     {
       "name": "core-js-3-only-polyfill/es6-array-last-index-of",
       "signals": "Array.prototype.lastIndexOf"
     },
     {
       "name": "core-js-3-only-polyfill/es6-array-map",
-      "signals": ""
+      "signals": "Array.prototype.map"
```

Comment:
example of the pattern matching not yielding results, but using source maps filled a gap.